src/main/java/org/folio/inventory/domain/instances/titles/PrecedingSucceedingTitle.java

```diff
@@ -44,4 +46,24 @@ public static PrecedingSucceedingTitle from(JsonObject rel, String title, String
       rel.getString(SUCCEEDING_INSTANCE_ID_KEY),
       title, hrid, identifiers);
   }
+
+  public JsonObject toPrecedingTitleJson() {
+    return toJson(PRECEDING_INSTANCE_ID_KEY, precedingInstanceId);
+  }
+
+  public JsonObject toSucceedingTitleJson() {
+    return toJson(SUCCEEDING_INSTANCE_ID_KEY, succeedingInstanceId);
+  }
+
+  private JsonObject toJson(String relatedInstanceIdKey, String relatedInstanceId) {
+    JsonObject json = new JsonObject();
+    includeIfPresent(json, "id", id);
+    includeIfPresent(json, TITLE_KEY, title);
+    includeIfPresent(json, HRID_KEY, hrid);
+    includeIfPresent(json, relatedInstanceIdKey, relatedInstanceId);
+    if (identifiers != null) {
+      json.put(IDENTIFIERS_KEY, identifiers);
+    }
+    return json;
```

Comment:
Personally, I might space this method out a little to create visual separation between steps.

```suggestion
    JsonObject json = new JsonObject();
    
    includeIfPresent(json, "id", id);
    includeIfPresent(json, TITLE_KEY, title);
    includeIfPresent(json, HRID_KEY, hrid);
    includeIfPresent(json, relatedInstanceIdKey, relatedInstanceId);
    
    if (identifiers != null) {
      json.put(IDENTIFIERS_KEY, identifiers);
    }
    
    return json;
```